components/definitions/rewards.d.ts

```diff
@@ -82,6 +83,36 @@ declare namespace Rewards {
     actions: any
   }
 
+  export interface MonthlyReport {
+    month: number
+    year: number
+    balance?: BalanceReport
+    transactions?: TransactionReport[]
+    contribution?: ContributionReport[]
+  }
+
+  export enum ReportType {
+    GRANT = 0,
+    AUTO_CONTRIBUTION = 1,
+    DEPOSIT = 2,
+    ADS = 3,
```

Comment:
for consistency, this should be `GRANT_AD`, however would read better as `AD_GRANTS`

Reply:
this is not new enum it's just copied from ledger

Reply:
Same as above

Reply:
synced with ledger